src/main/scala/io/iohk/ethereum/jsonrpc/IeleJsonMethodsImplicits.scala

```diff
@@ -0,0 +1,104 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.jsonrpc.EthService._
+import io.iohk.ethereum.jsonrpc.JsonRpcController.JsonDecoder
+import io.iohk.ethereum.jsonrpc.JsonRpcErrors.InvalidParams
+import io.iohk.ethereum.jsonrpc.PersonalService.{InvalidAddress, SendIeleTransactionRequest}
+import org.json4s.JsonAST.{JArray, JObject, JString, JValue}
+
+object IeleJsonMethodsImplicits extends JsonMethodsImplicits {
+
+  def extractIeleCall(obj: JObject): Either[JsonRpcError, IeleCallTx] = {
+    def toEitherOpt[A, B](opt: Option[Either[A, B]]): Either[A, Option[B]] =
+      opt.map(_.right.map(Some.apply)).getOrElse(Right(None))
+
+    for {
+      from <- toEitherOpt((obj \ "from").extractOpt[String].map(extractBytes))
+      to <- toEitherOpt((obj \ "to").extractOpt[String].map(extractBytes))
+      gas <- optionalQuantity(obj \ "gas")
+      gasPrice <- optionalQuantity(obj \ "gasPrice")
+      value <- optionalQuantity(obj \ "value")
+      functionName = (obj \ "functionName").extractOpt[String]
```

Comment:
we've previously used the name "function" for this field instead of "functionName". If you don't have a strong opinion, it may be simpler to do the same thing here.